aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,7 +37,23 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
-By default, CheckBox has built-in `pad` to create space between the CheckBox and FormField border. When using CheckBox outside of a FormField, apply `pad="none"`.
+A checkbox might be used outside of a form field when:
+1. Serving as a UI control showing or hiding page content, such as revealing additional content or inputs which are only relevant when a particular feature is enabled.
+2. Scenario's in which having a formfield label in addition to the checkbox label would is to redundant for the user. 
+2. There is no additional information or help text necessary for the user to understand what is being asked of them.
```

Comment:
I think this can go now that we have the new #2.

```suggestion
```

Reply:
opps yes forgot that commit